scripts/brp-strip-static-archive

```diff
@@ -5,16 +5,18 @@ if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
 fi
 
 STRIP=${1:-strip}
+NCPUS=${2:-1}
 
 case `uname -a` in
 Darwin*) exit 0 ;;
 *) ;;
 esac
 
 # Strip static libraries.
-for f in `find "$RPM_BUILD_ROOT" -type f -a -exec file {} \; | \
-        grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
+for f in `find "$RPM_BUILD_ROOT" -type f | \
+	grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug" | \
+	xargs -r -P$NCPUS -n16 file | sed 's/:  */: /' | \
```

Comment:
why `-n16`

Reply:
man xargs says:
Use the -n option or the -L option with -P; otherwise chances are that only one exec will be done.
While the number was chosen by our kernel folks I guess it offers a good compromise between starting all the helpers for each file and preventing proper parallelism due to doing to much at once.